newIDE/app/src/MainFrame/index.js

```diff
@@ -1059,10 +1059,10 @@ const MainFrame = (props: Props) => {
       // Try to find an autosave (and ask user if found)
       try {
         await delay(150);
-        const autoSaveFileMetadata = await checkForAutosave();
         let content;
         let openingError: Error | null = null;
         try {
+          const autoSaveFileMetadata = await checkForAutosave();
```

Comment:
Can this throw? In which case, should we ignore the result and consider there is no auto save?

We can also consider that checkForAutosave is responsible for never throwing (and so, return false)

Reply:
It was failing when developing the feature and the error was handled but the loader was still displayed, preventing the use of the app so I put it in the try clause. But I can also make sure that it return false if there is an error